CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,103 @@
+
+## Our Purpose
+
+The PureScript community spaces are places to discuss the PureScript programming language. In these spaces:
+
+* We talk about the language, its related libraries and infrastructure, and our experiences using it.
+* We help PureScript programmers learn the language and we ask for help becoming stronger programmers ourselves. 
+* We discuss general functional programming concepts and other parts of programming, type theory, and mathematics that are relevant to Purescript.
+* We organize open-source work, meetups, hackathons, and other community events.
+* We discuss what we can do to grow an inclusive, friendly, and respectful community.
+
+## Unacceptable Behavior
+
+The following types of behavior are unacceptable in the PureScript community and constitute violations of this code of conduct.
+
+### Abusive behavior
+
+- **Harassment**: making offensive comments related to age, body size, disability, ethnicity, sex characteristics, gender identity and expression, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation. This also includes the use of sexualized language or imagery and unwelcome sexual or romantic attention.
+- **Threats**: threatening someone physically, verbally, or otherwise, such as threatening to publicize sensitive information about someone's personal life.
+
+### Unwelcoming behavior
+
+- **Prejudice and blatant -isms**: saying things that are explicitly racist, sexist, homophobic, transphobic, ageist, or otherwise harmful expressions of prejudice and bias. For example: arguing that some people are less intelligent because of their gender, race, or religion. Subtle expressions of harmful prejudice can be small mistakes, and aren't considered violations. However, repeating something after it has been pointed out to you that it breaks this rule, or arguing with someone who has pointed that out to you, is considered unwelcoming behavior and is not allowed.
+- **Maliciousness towards other members**: deliberately attempting to make others feel bad, name-calling, or singling others out for derision. For example: telling someone they're a poor programmer.
+- **Being especially unpleasant**: for example, if we've received several reports of annoying, rude, or especially distracting behavior.
+
+The rule around prejudice and blatant -isms is especially complicated because not everyone agrees on what constitutes a violation. Members of the PureScript community live in disparate cultures and speak different languages. Harmful prejudices are baked into society in ways that can make them hard to recognize, and not everyone experiences prejudice in the same way: subtle ageism won't hurt someone who is young in the same way it hurts someone who is older.
+
+If you are told that a symbol, phrase, or message is unwelcome, please apologize and move on. It's not a big deal for community members to make a mistake from time to time.
```

Comment:
> ‘If you are told’ by a moderator

This is the correct reading. I'd assume that in practice, for particularly obvious cases, members of the community might note some language or action isn't generally considered acceptable here. But it's only when you are told _by a moderator_ that the rule about not arguing comes into play.

There is no intent to empower individuals to arbitrarily decide on new forbidden language or behavior, if only because it then becomes extremely difficult to moderate. But individuals can certainly suggest or ask for certain things to be moderated, in which case it is still up to the moderators to make any 'official' decision on it.

Reply:
I've updated this! Thanks for pointing it out.